src/app/components/navbar/objects/board.object.ts

```diff
@@ -1,11 +1,21 @@
+import {Window} from '../../grid/objects/window.object';
+
 export class Board{
+    id:number;
     title:string;
     dateCreated: string;
     icon: string;
+    windows:Array<Window>;
 
-    constructor(){
-        this.title = "BOARD TITLE";
+    constructor(title:string, icon: string){
+        this.id = Math.floor(Math.random() * 0) + 100;
```

Comment:
The id property of the boards is pretty useless until we actually have a database set up that will number and keep track of the id's for us.